asab/library/providers/zookeeper.py

```diff
@@ -44,6 +57,9 @@ async def read(self, path):
 
 
 	async def list(self, path):
+		if self.Zookeeper is None:
+			return
+
 		recursive = path.endswith("*")
```

Comment:
@bochkver 
Is the asterisk really needed? Could the method take boolean `recursive` as a second argument? When I try to use it, I have to append asterisk that is removed here...

Reply:
Yep, `recursive` as a second argument will be better (same fix is needed in `providers/filesystem.py`)

Reply:
:slightly_smiling_face: perfect